tests/framework/validators/UniqueValidatorTest.php

```diff
@@ -136,4 +138,28 @@ public function testValidateCompositeKeys()
         $val->validateAttribute($m, 'id');
         $this->assertFalse($m->hasErrors('id'));
     }
+
+    public function testValidateTargetClass()
+    {
+        $val = new UniqueValidator([
+            'targetClass' => Category::className(),
+            'targetAttribute' => ['name'],
```

Comment:
could you explain the semantics behind this validation? why would I validate uniqueness of item name against categories?